prometheus-ksonnet/lib/prometheus.libsonnet

```diff
@@ -65,7 +65,7 @@
   local pvc = $.core.v1.persistentVolumeClaim,
 
   prometheus_pvc::
-    if ! $._config.stateful
+    if !$._config.stateful
```

Comment:
I believe this is a syntax error introduce by an old version of jsonnet fmt.  See https://github.com/google/jsonnet/issues/572

There is a workaround using brackets on that issue.